### PR TITLE
build: build freetype/harfbuzz/fribidi and enable text support on Windows

### PR DESCRIPTION
**Pull request by mbouron**

### CODE REVIEW

File: configure.py [hunk 1]
```diff
@@ -113,6 +128,9 @@ def _get_external_deps(args):
         deps.append("ffmpeg")
         deps.append("sdl2")
         deps.append("glslang")
+        deps.append("harfbuzz")
+        deps.append("freetype")
+        deps.append("fribidi")
```
Comment (ubitux) [inline configure.py:133]: nit: freetype, harfbuzz, fribidi for consistency with the other order you picked?

**Review by ubitux — COMMENTED** [review]
I believe It's missing entries in the `clean` rule.

File: configure.py (hunk 1 above)
Comment (mbouron) [reply configure.py]: Done.

**Comment by mbouron** [comment]
> I believe It's missing entries in the `clean` rule.

Done.

**Review by ubitux — APPROVED** [review]

**Pull request merged**